lib/graphql/schema/build_from_ast.rb

```diff
@@ -0,0 +1,260 @@
+module GraphQL
+  class Schema
+    # You can use the result of {GraphQL.parse} to make a schema.
+    # This schema is missing some important details like
+    # `resolve` functions, but it does include the full type system,
+    # so you can use it to validate queries.
+    module BuildFromAST
+      extend self
+
+      class InvalidDocument < Error; end;
```

Comment:
Can we end this classname with `Error`? that would make it consistent with the rest of the errors in the library.